From 73b535d23d98bd3bdc31a27da26222e2e56166ac Mon Sep 17 00:00:00 2001
From: Karl Williamson <khw@cpan.org>
Date: Tue, 30 Jun 2020 13:58:50 -0600
Subject: [PATCH] ext/XS-APItest/t/utf8_warn_base.pl: Fix a couple tests
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

These had invalid values, which didn't show up execpt on EBCDIC

Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 ext/XS-APItest/t/utf8_warn_base.pl | 2 --
 1 file changed, 2 deletions(-)

diff --git a/ext/XS-APItest/t/utf8_warn_base.pl b/ext/XS-APItest/t/utf8_warn_base.pl
index d86871cd0f..a0f732282e 100644
--- a/ext/XS-APItest/t/utf8_warn_base.pl
+++ b/ext/XS-APItest/t/utf8_warn_base.pl
@@ -486,7 +486,6 @@ my @tests;
               : I8_to_native(
                 "\xff\xa7\xbf\xbf\xbf\xbf\xbf\xbf\xbf\xbf\xbf\xbf\xbf\xbf"),
               0x7FFFFFFFFFFFFFFF,
-              (isASCII) ? 1 : 2,
             ],
             [ "first 64 bit code point",
               (isASCII)
@@ -525,7 +524,6 @@ my @tests;
                     I8_to_native(
                     "\xff\xa0\xa0\xa0\xa0\xa0\xa1\xa0\xa0\xa0\xa0\xa0\xa0\xa0"),
                     0x800000000,
-                      40000000
                 ],
                 [ "requires at least 32 bits",
                     I8_to_native(
-- 
2.25.4

